Wikipedia/Code/MWKSection.m

```diff
@@ -244,7 +244,7 @@ - (nullable NSString *)summary {
     }] componentsJoinedByString:@" "] wmf_summaryFromText];
 }
 
-static NSString *const WMFSectionDisambiguationTitlesXPathSelector = @"//div[@class='hatnote']//a/@href";
+static NSString *const WMFSectionDisambiguationTitlesXPathSelector = @"//div[contains(@class, 'hatnote')]//a/@href";
 
 - (nullable NSArray<NSURL *> *)disambiguationURLs {
```

Comment:
iirc `disambiguationURLs` and `pageIssues` are only used by the footer menu transform anyway (they do hand-off to native land when tapped, but shouldn't make much difference if they figure out how to get the disambig urls & page-issues strings)